### PR TITLE
log: delete lightning's redact package

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
lightning and br's redact package are duplicated.

### What is changed and how it works?
replace lightning's redact package with br's.

### Check List 

Tests 

 - Unit test
 - Integration test

### Release Note

 - No release note

### CODE REVIEW

File: pkg/lightning/backend/tidb/tidb.go [hunk 1]
```diff
@@ -59,7 +60,7 @@ type tidbRows []tidbRow
 // MarshalLogArray implements the zapcore.ArrayMarshaler interface
 func (rows tidbRows) MarshalLogArray(encoder zapcore.ArrayEncoder) error {
 	for _, r := range rows {
-		encoder.AppendString(string(r))
+		encoder.AppendString(redact.String(string(r)))
```
Comment (kennytm) [inline pkg/lightning/backend/tidb/tidb.go:63]: there's a slight difference between this and the original `ZapRedactArray`. 

when redaction is on, the marshal output in `ZapRedactArray` was always `?`, while here we get `?,?,?,?,?`, and thus "leaking" the number of columns. not sure if it is considered sensitive though.

File: pkg/logutil/logging.go [hunk 2]
```diff
@@ -186,3 +186,11 @@ func Keys(keys [][]byte) zap.Field {
 func ShortError(err error) zap.Field {
 	return zap.String("error", err.Error())
 }
+
+// Any constructs a field that carries an interface{}.
```
Comment (kennytm) [inline pkg/logutil/logging.go:190]:
```suggestion
// Any constructs a redacted field that carries an interface{}.
```

maybe name it `RedactAny`.

**Review by kennytm — COMMENTED** [review]
rest LGTM

File: pkg/lightning/backend/tidb/tidb.go (hunk 1 above)
Comment (lichunzhu) [reply pkg/lightning/backend/tidb/tidb.go:63]: Table structure and number of columns seem not sensitive. PTAL @overvenus

**Pull request merged**